src/sagemaker_containers/_env.py

```diff
@@ -757,6 +779,16 @@ def output_data_dir(self):  # type: () -> str
         """
         return self._output_data_dir
 
+    @property
+    def output_intermediate_dir(self):  # type: () -> str
+        """The dir to write intermediate output artifacts for them to be synced to S3.
+        The directory special behavior is to move artifacts from the training instance to
```

Comment:
Wording is confusing. Maybe:

"Any files written to this directory will be uploaded to S3 by a background process while training is in progress."